src/clusterfuzz/_internal/bot/fuzzers/syzkaller/config.py

```diff
@@ -64,6 +64,11 @@ def generate(serial,
     data['target'] = 'linux/amd64'
     data['disable_syscalls'] = ['openat$vhost_vsock']
     data['sandbox'] = 'none'
+    # Cuttlefish pass kernel log as console input in Syzkaller config.
+    device = {}
+    device['serial'] = serial
+    device['console'] = '/home/vsoc-01/cuttlefish_runtime.1/kernel.log'
```

Comment:
is this path guaranteed to be stable? what is the ".1" in "cuttlefish_runtime.1" ? 

probably add a comment to explain this.